kotsadm/web/src/components/apps/AppDetailPage.jsx

```diff
@@ -137,7 +155,7 @@ class AppDetailPage extends Component {
    * @return {undefined}
    */
   refetchGraphQLData = () => {
```

Comment:
A note. We should change this to just be `refetchData` once we have the list apps query done since GQL will be gone